mainnet/run-observer.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-docker run --mount type=bind,source=${OBSERVER_DIR}/db,destination=/go/elrond-go/cmd/node/db --mount type=bind,source=${OBSERVER_DIR}/logs,destination=/go/elrond-go/cmd/node/logs --mount type=bind,source=${OBSERVER_DIR}/config,destination=/config --publish ${P2P_PORT}:37373 --network=elrond-squad --ip=${IP} --name squad-${SHARD} elrondnetwork/elrond-node-obs:v1.1.10 \
+docker run --mount type=bind,source=${OBSERVER_DIR}/db,destination=/go/elrond-go/cmd/node/db --mount type=bind,source=${OBSERVER_DIR}/logs,destination=/go/elrond-go/cmd/node/logs --mount type=bind,source=${OBSERVER_DIR}/config,destination=/config --publish ${P2P_PORT}:37373 --network=elrond-squad --ip=${IP} --name squad-${SHARD} elrondnetwork/${NODE_TAG} \
```

Comment:
From the README, the user doesn't know that it should also set `NODE_TAG` (when using the basic approach, not the docker compose):

```
export SHARD=2
export DISPLAY_NAME="MyObservingSquad-2"
export OBSERVER_DIR=~/MyObservingSquad/node-2
export P2P_PORT=10002
export IP=10.0.0.4
./mainnet/run-observer.sh
```

Perhaps update the readme?

Reply:
reverted